src/main/java/helloworld/HelloWord.java

```diff
@@ -8,19 +8,25 @@
 import helloworld.s3_lambda_sqs.Sqs;
 
 import static helloworld.constant.Constants.BUCKET_NAME_OUTPUT;
+import static helloworld.constant.Constants.BUCKET_OUTPUT_KEY;
 import static helloworld.constant.Constants.FILE_NAME;
+import static helloworld.constant.Constants.LAMBDA_FUNCTION_INPUT_HANDLER;
 import static helloworld.constant.Constants.LAMBDA_FUNCTION_INPUT_NAME;
+import static helloworld.constant.Constants.LAMBDA_FUNCTION_OUTPUT_HANDLER;
 import static helloworld.constant.Constants.LAMBDA_FUNCTION_OUTPUT_NAME;
+import static helloworld.constant.Constants.LAMBDA_FUNCTION_ROLE;
 import static helloworld.constant.Constants.QUEUE_OUTPUT_NAME;
 
 public class HelloWord {
     public static void main(String[] args) {
         S3Bucket bucket = new S3Bucket(AmazonS3ClientBuilder.standard().withRegion(Regions.EU_WEST_1).build(), BUCKET_NAME_OUTPUT);
         Sqs output_queue = new Sqs(AmazonSQSClientBuilder.defaultClient(), QUEUE_OUTPUT_NAME);
+        Lambda lambdaOutput = Lambda.builder().lambdaName(LAMBDA_FUNCTION_OUTPUT_NAME).role(LAMBDA_FUNCTION_ROLE).s3BucketName(BUCKET_NAME_OUTPUT).s3BucketKey(BUCKET_OUTPUT_KEY).handler(LAMBDA_FUNCTION_OUTPUT_HANDLER).build();
```

Comment:
With method chaining, when there are many of them (more than 120 characters, the limit that your IntelliJ shows a line in the vertical), it's a good practice to break them down. It improves a lot the readability ;) 
```suggestion
        Lambda lambdaOutput = Lambda.builder()
                .lambdaName(LAMBDA_FUNCTION_OUTPUT_NAME)
                .role(LAMBDA_FUNCTION_ROLE)
                .s3BucketName(BUCKET_NAME_OUTPUT)
                .s3BucketKey(BUCKET_OUTPUT_KEY)
                .handler(LAMBDA_FUNCTION_OUTPUT_HANDLER)
                .build();
```

Reply:
what u mean?